.github/workflows/debian.yml

```
@@ -26,7 +26,7 @@ jobs:
         run: |
           apt-get update
           apt-get install --no-install-recommends -y \
-              build-essential librocksdb-dev cython3 \
+              build-essential librocksdb-dev cython3 pkg-config \
               python3-dev python3-pip python3-pytest \
```

What you need is the python library, sadly the tests still fail
```suggestion
              build-essential librocksdb-dev cython3 \
              python3-dev python3-pip python3-pkgconfig python3-pytest \
```

---

Thanks.